data-prepper-plugins/peer-forwarder/README.md

```diff
@@ -35,16 +52,26 @@ prepper:
 * `ssl` => Default is ```false```.
 * `sslKeyCertChainFile` => Should be provided if ```ssl``` is set to ```true```
 
-## Operating a Cluster with DNS Discovery
-DNS discovery is recommended when scaling out a Data Prepper cluster. The core concept is to configure a DNS provider to return a list of Data Prepper hosts when given a single domain name.
+## Metrics
 
-### With Kubernetes (Recommended)
-[Kubernetes](https://kubernetes.io/) is the recommended approach to managing a Data Prepper cluster. In addition to handling tasks like failure detection and host replacement, it also maintains an internal DNS service for pod discovery. [Headless services](https://kubernetes.io/docs/concepts/services-networking/service/#headless-services) allow for a single service address to map to all Data Prepper pods. This DNS entry is automatically kept up-to-date as pods are created/replaced/destroyed.
+Besides common metrics in [AbstractPrepper](https://github.com/opendistro-for-elasticsearch/data-prepper/blob/master/data-prepper-api/src/main/java/com/amazon/dataprepper/model/prepper/AbstractPrepper.java), peer-forwarder introduces the following custom metrics.
 
-See the /examples/dev/k8s directory for a working example using minikube.
+### Timer
 
-### With a custom DNS server
-A DNS server (like [dnsmasq](http://www.thekelleys.org.uk/dnsmasq/doc.html)) can be configured to maintain a list of Data Prepper hosts via config files. Data Prepper hosts must be configured to use the custom DNS server as their DNS provider. The list of hosts must be manually updated whenever a new Data Prepper host is created. See the [/examples/dev/dns directory](https://github.com/opendistro-for-elasticsearch/data-prepper/tree/master/examples/dev/dns) for a set of sample dnsmasq configuration files.
+- `forwardRequestLatency`: measures latency in forwarding each request.
 
-### With Amazon Route 53 Private Hosted Zones
-[Private hosted zones](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/hosted-zones-private.html) enable Amazon Route 53 to "respond to DNS queries for a domain and its subdomains within one or more VPCs that you create with the Amazon VPC service." Similar to the custom DNS server approach, except that Route 53 maintains the list of Data Prepper hosts. Suffers from the same drawback in that the list must be manually kept up-to-date.
+### Counter
+
+- `forwardRequestSuccess`: measures number of successfully sent requests.
+- `forwardRequestErrors`: measures number of failed requests.
+
+### Gauge
+
+- `peerEndpoints`: measures number of dynamically discovered peer data-prepper endpoints. For `static` mode, the size is fixed.  
+
+## Developer Guide
+
+This plugin is compatible with Java 14. See 
```

Comment:
Is 14 the minimum version? I think I was using Java 11 in my IDE :-/

Reply:
It is highest version I think. For minimum Java 8 will also work probably.

Reply:
Would suggest listing the minimum version instead of max, otherwise users might think they _need_ to have Java 14 to run the app.

Reply:
I checked that we do need Java 14 for ES sink. For other plugins it might vary. So I will leave it as it is.

Reply:
I'm running test instances with Java 11 (so is the CFN template), where are you seeing a dependency on Java 14? 🤔

Reply:
Interesting...I doubt if it is due to ES test framework